src/middleware/queries.ts

```diff
@@ -273,11 +273,11 @@ export function proposals(
 /**
  * @hidden
  *
- * Get the tickers of all the tokens for which the passed DID is a trusted claim issuer
+ * Get the tickers of all the Assets for which the passed DID is a trusted claim issuer
  */
-export function tokensByTrustedClaimIssuer(
-  variables: QueryTokensByTrustedClaimIssuerArgs
-): GraphqlQuery<QueryTokensByTrustedClaimIssuerArgs> {
+export function assetsByTrustedClaimIssuer(
```

Comment:
I think this function (and the other one) should retain their original names, since they are named after their underlying GQL query